docker-compose.prod.yml

```
@@ -0,0 +1,124 @@
+# Example compose deployment
+
+x-deploy: &default-deploy
+  restart_policy:
+    condition: on-failure
+    delay: 5s
+    max_attempts: 3
+    window: 120s
+  update_config:
+    order: start-first
+
+version: '3.7'
+
+services:
+  caddy:
+    image: caddy:2
+    volumes:
+      - /data/caddy:/data/caddy
+    ports:
+      - 80:80
+      - 443:443
+    command: caddy reverse-proxy --to firezone:4000 --from ${EXTERNAL_URL?err}
+    deploy:
+      <<: *default-deploy
+
+  firezone:
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    image: fz
```

```suggestion
    image: firezone/firezone
```

Waiting on docker hub repo.

---

Created